assets/css/styleguide.scss

```diff
@@ -308,8 +304,15 @@ a#skipnav {
   }
 
   p, a {
```

Comment:
I feel like this is a particularly bad rule to put in styleguide.scss. I get how we're out of time so can you put a comment here saying to investigate further? Like this;

```
TODO investigate why this is required in styleguide.scss.
```